website/docs/r/private_key.html.md

```diff
@@ -24,43 +23,71 @@ state and does not create any external managed resources.
 ## Example Usage
 
 ```hcl
+// ECDSA key with P384 elliptic curve
```

Comment:
Nit: Within Terraform HCL, `#` is more common for comments. Refer also to [Terraform configuration language](https://www.terraform.io/language/syntax/configuration#comments) documentation:

> The # single-line comment style is the default comment style and should be used in most cases. Automatic configuration formatting tools may automatically transform // comments into # comments, since the double-slash style is not idiomatic.

```suggestion
# ECDSA key with P384 elliptic curve
```